lib/inets/src/http_client/httpc_handler.erl

```diff
@@ -1300,15 +1301,15 @@ is_keep_alive_enabled_server("HTTP/1.0",
 is_keep_alive_enabled_server(_,_) ->
     false.
 
-is_keep_alive_connection(Headers, #session{client_close = ClientClose}) ->
+is_keep_alive_connection(Headers, #session{client_close = ClientClose}) when is_boolean(ClientClose) ->
```

Comment:
I think this would be better by setting the default in the record to false. There is no use for this field to be undefined!